sequencing/ATACSeq/Rscripts/collateSexChecks.r

```diff
@@ -58,18 +89,19 @@ a2 <- ggplot(xPeaksWide, aes(x=XIST, y=YTot, color=sex)) +
 figa<-ggarrange(a1, a2,  
           ncol = 2, nrow = 1)
 
-pdf(file.path(dataPath, "3_aligned", "QCOutput", "sexPredictions.pdf"), width = 10, height = 5)
+pdf(file.path(dir, "3_aligned", "QCOutput", "sexPredictions.pdf"), width = 10, height = 5)
 print(figa)
 dev.off()
 		  
 ## predict sex
-
+#For Shifted Tag Align mode
 sexPredict<-data.frame("sampleID" = gsub(".chrX.tn5.tagAlign.gz", "", xPeaksWide$V18),"LabelledSex" = xPeaksWide$sex, "FIRREratio" = ratioF > thresF,"XISTratio" = ratioX > thresX)
+
 ## TRUE means male
 sexPredict[sexPredict == TRUE]<-"M"
 sexPredict[sexPredict == FALSE]<-"F"
```

Comment:
Just a quick check, should this be:
```R
sexPredict[sexPredict[["LabelledSex]] == TRUE]<-"M"
sexPredict[sexPredict[["LabelledSex]] == FALSE]<-"F"
```

Reply:
This only matters if there are values in `sexPredict` outside of the LabelledSex column that are `1`, `TRUE` or `FALSE`. Not sure if this is the case or not.

Reply:
Ahh, so `FIRREratio` and `XISTratio` are supposed to be columns containing `M` and `F`?

Reply:
Yes, as in  `sexPredict<-data.frame("sampleID" = xPeaksWide$sampleID,"LabelledSex" = xPeaksWide$sex, "FIRREratio" = ratioF > thresF,"XISTratio" = ratioX > thresX)`  FIRREratio and XISTratio will be either TRUE or FALSE